Libraries/Types/CoreEventTypes.js

```diff
@@ -152,6 +152,13 @@ export type FocusEvent = SyntheticEvent<
   |}>,
 >;
 
+export type KeyEvent = SyntheticEvent<
+  $ReadOnly<{|
+    characters: string,
```

Comment:
FYI that @harinikmsft has a doc where there were some ideas of what event shape for xplat keyboard events should look like. Would be great to keep alignment between platforms. https://github.com/microsoft/react-native-windows/blob/master/vnext/proposals/active/keyboard-reconcile-desktop.md

Reply:
Decided to make the API more similar to the react-native-windows one and pass each of these flags through to the client like how win does :)